cirq-core/cirq/optimizers/merge_interactions_test.py

```diff
@@ -30,13 +30,19 @@ def assert_optimizes(before: cirq.Circuit, expected: cirq.Circuit):
         cirq.merge_single_qubit_gates_into_phased_x_z,
         cirq.EjectPhasedPaulis().optimize_circuit,
         cirq.EjectZ().optimize_circuit,
-        cirq.DropNegligible().optimize_circuit,
-        cirq.DropEmptyMoments().optimize_circuit,
     ]
     for post in followup_optimizations:
         post(actual)
         post(expected)
 
+    followup_optimizations_new: List[cirq.TRANSFORMER] = [
+        cirq.drop_negligible_operations,
+        cirq.drop_empty_moments,
+    ]
+    for post_new in followup_optimizations_new:
+        actual = post_new(actual).unfreeze(copy=False)
+        expected = post_new(expected).unfreeze(copy=False)
```

Comment:
I'd suggest using the "transformer" terminology here:
```suggestion
    followup_transformers: List[cirq.TRANSFORMER] = [
        cirq.drop_negligible_operations,
        cirq.drop_empty_moments,
    ]
    for transform in followup_transformers:
        actual = transform(actual).unfreeze(copy=False)
        expected = transform(expected).unfreeze(copy=False)
```